docs/runwhen/concepts.md

```diff
@@ -0,0 +1,101 @@
+# RunWhen Concepts
+- [RunWhen Concepts](#runwhen-concepts)
+- [Runwhen Local](#runwhen-local)
+  - [CheatSheet Generator](#cheatsheet-generator)
+  - [Uploading Cluster Topology to the Platform](#uploading-cluster-topology-to-the-platform)
+- [CodeCollections](#codecollections)
+- [CodeBundles](#codebundles)
+
+# Runwhen Local
+- [source-code](https://github.com/runwhen-contrib/runwhen-local)
+- [Helm Chart](https://github.com/runwhen-contrib/helm-charts/tree/main/charts/runwhen-local)
+- [Upstream docs](https://docs.runwhen.com/public/v/runwhen-local/)
+  
+RunWhen Local has two core functions:
+- Generate remediation scripts / CheatSheets from included templates for your local cluster
+- Upload Cluster Topology to the RunWhen Platform
+
+## CheatSheet Generator
+At the moment RunWhen Local **does not posses the ability to discover issues** in
+your cluster and suggest mitigation runbooks / codebundles. 
+
+**However, it discovers your kubernetes resources and object names.**
+Using which, it generates a wide set of runbooks for you, if you already know the
+root cause. These runbooks contain documentation and pastable shell script
+snippets for the searched issue. These scripts / cheatsheet are already pre-templated
+with your namespaces and kubernetes resource names.
+
+This collection of cheatsheets / runbooks, although not exhaustive, covers a significant portion
+of recurring issues and healthcheck failures and can be useful to SREs for quick
+resolution of incidents.
+
+[Upstream Examples](https://docs.runwhen.com/public/v/runwhen-local/user-guide/features/user_guide-feature_overview)
+
+## Uploading Cluster Topology to the Platform
+The second core function of runwhen-local is to upload cluster topology to the
+runwhen platform so you can visualize the cluster workload map from a configured
+runwhen workspace.
+
+- First, follow documentation at [Upload to RunWhen Platform](https://docs.runwhen.com/public/v/runwhen-local/user-guide/features/upload-to-runwhen-platform#upload-from-the-cli)
+  - To generate the `uploadInfo.yaml` file
+- Next, take the yaml object and copy over it's contents to `uploadInfo:[]` section
+of the helm [`values.yaml` file](https://github.com/runwhen-contrib/helm-charts/blob/main/charts/runwhen-local/values.yaml#L121)
+- Once configured it should look like this:
+  ```YAML
+  uploadInfo:
+    workspaceName: <your-workspace-name>
+    token: <your token> # Do NOT add token and commit to git
+    workspaceOwnerEmail: tester@my-company.com
+    papiURL: https://papi.beta.runwhen.com
+    defaultLocation: location-01-us-west1 # available runwhen locations
+  ```
+- You should pass the token from helm cli, to ensure you are not leaking the token via git
```

Comment:
we have our own script to install runwhen-local using makefile, we can say to install runwhen-local run: 

`make setup-runwhen`